app/services/ui_notifications/hosts/missing_owner.rb

```diff
@@ -0,0 +1,29 @@
+module UINotifications
+  module Hosts
+    class MissingOwner < UINotifications::Base
+      private
+      def create
+        add_notification if update_notifications.zero?
+      end
+
+      def update_notifications
+        blueprint.notifications.
+          where(subject: subject).
+          update_all(expired_at: blueprint.expired_at)
+      end
+
+      def add_notification
+        Notification.create!(
+          initiator: initiator,
+          audience: 'admin',
```

Comment:
nitpick, `Notification::AUDIENCE_ADMIN` to be resilient to changes and use the frozen string